crates/router/src/connector/dummyconnector/transformers.rs

```diff
@@ -4,11 +4,14 @@ use serde::{Deserialize, Serialize};
 use url::Url;
 
 use crate::{
+    connector::utils,
     core::errors,
     services,
     types::{self, api, storage::enums},
 };
 
+type Error = error_stack::Report<errors::ConnectorError>;
```

Comment:
Nitpick: The name of the alias is too generic
```suggestion
type ConnectorError = error_stack::Report<errors::ConnectorError>;
```
Update it as per suggestion or remove it altogether since it doesn't seem like it's needed at more than one place.